routes/product.routes.js

```diff
@@ -0,0 +1,108 @@
+const { Router } = require('express') 
+const router = Router()
+const productModel = require('../models/product')
+const lastItem = require('../utils/lastItem')
+
+router.get(
+  '/new',
+  async (req, res) => {
+    try {
+      const { dataModel } = await lastItem(productModel, '', '', 6)
+
+      let products = await dataModel.map(item => ({
```

Comment:
do we need pagination here?

Reply:
no, in the func above, the limit number of elements is specified